segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -146,16 +173,16 @@ public void performNextWrite() {
         }
 
         Timer timer = new Timer();
-        storeAppend(append).whenComplete((v, e) -> {
-            handleAppendResult(append, e);
-            if (e == null) {
-                WRITE_STREAM_SEGMENT.reportSuccessEvent(timer.getElapsed());
-            } else {
-                WRITE_STREAM_SEGMENT.reportFailEvent(timer.getElapsed());
-            }
-        }).whenComplete((v, e) -> {
-            append.getData().release();  
-        });
+        storeAppend(append)
```

Comment:
I found it pretty hard to follow the flow here since some callbacks are on their own lines and some are not. I standardized them all to start on new lines, so that the IDE can properly format them.